src/accessories/TokenArranger.ttslua

```diff
@@ -20,6 +20,8 @@ inputParameters.tab             = 2
 
 local latestLoad = "XXX"
 local updating = false
+local percentage = false
+local cumulative = false
```

Comment:
Imo it would be better if this was combined into one variable, for example:
disabled (no percentages): `showPercentages = false`
enabled (specific values): `showPercentages = "specific"`
enabled (cumulative values): `showPercentages = "cumulative"`

Reply:
Excellent idea!  Implemented